APITesters/SwiftAPITester/SwiftAPITester.xcodeproj/project.pbxproj

```diff
@@ -62,18 +100,42 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		2D6F3871270E58DB002C9987 /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+			);
+			name = Frameworks;
+			sourceTree = "<group>";
+		};
+		2DD778D1270E233F0079CBD4 /* SwiftAPITesterIOS */ = {
+			isa = PBXGroup;
+			children = (
+				2DD778D2270E233F0079CBD4 /* AppDelegate.swift */,
+				2DD778D4270E233F0079CBD4 /* SceneDelegate.swift */,
+				2DD778D6270E233F0079CBD4 /* ViewController.swift */,
+				2DD778D8270E233F0079CBD4 /* Main.storyboard */,
+				2DD778DB270E23400079CBD4 /* Assets.xcassets */,
+				2DD778DD270E23400079CBD4 /* LaunchScreen.storyboard */,
+				2DD778E0270E23400079CBD4 /* Info.plist */,
+			);
+			path = SwiftAPITesterIOS;
+			sourceTree = "<group>";
+		};
 		A55F62AC26EAFFD200A1B466 = {
 			isa = PBXGroup;
 			children = (
 				A55F62B726EAFFD200A1B466 /* SwiftAPITester */,
+				2DD778D1270E233F0079CBD4 /* SwiftAPITesterIOS */,
```

Comment:
I'm in the process of keeping this new target as the only one by making it cross-platform, to remove the other one :)

Reply:
awesome!